### ISSUE TITLE
Labels/fields - placement

### ISSUE THREAD

**Issue by StaggerLeee** [pods-framework/pods#2797]
Tested with Pods 3.0 Beta too.
When you drag Pods metabox to the right (post edit sidebar) it looks so unnatural and strange. Table HTML formatting is not for sidebar.
Label takes most of the box width and input fields are so small. In some sense it cannot be used side-right when situation is like this.

Can something be done about it ? Labels above option. Look how Advanced Custom fields manage it. It is GPL, you can borrow code. If it is not to late for it now.

![2015-03-15_152655](https://cloud.githubusercontent.com/assets/7382616/6656351/c02c6af2-cb27-11e4-9454-93270df167f5.jpg)

**Comment by StaggerLeee** [comment]
Meta.php file --> line 1130. Should it have "td" instead of "th" ?

Some quick fix for people that need it for now:

Labels above fields:
```
/**
 * Snippet Name: Plain CSS in admin area header
 * Snippet URL: http://www.wpcustoms.net/snippets/custom-css-in-admin-area-header/
 */
 function my_custom_backend_css() {
  echo '<style>
#side-sortables .form-table th {
box-sizing: border-box;
display: block;
width: 100%;
padding: 0;
margin: 0;
}
#side-sortables .form-table tr.pods-field > td {
display: block;
line-height: inherit;
padding: 0;
margin: 0;
}
  </style>';
}
add_action('admin_head', 'my_custom_backend_css');
```

Labels inline with fields:
```
/**
 * Snippet Name: Plain CSS in admin area header
 * Snippet URL: http://www.wpcustoms.net/snippets/custom-css-in-admin-area-header/
 */
 function my_custom_backend_css() {
  echo '<style>
#side-sortables .form-table th {
width: auto;
}
  </style>';
}
add_action('admin_head', 'my_custom_backend_css');
```
![2015-03-15_155140](https://cloud.githubusercontent.com/assets/7382616/6656436/59a972e4-cb2b-11e4-9b2c-5d0a338e47d0.jpg)
![2015-03-15_155224](https://cloud.githubusercontent.com/assets/7382616/6656441/5d3e9f56-cb2b-11e4-8f18-669471283917.jpg)

**Comment by StaggerLeee** [comment]
Sorry, it nened to be more targeted to not ruin other things in backend:

- #side-sortables .pods-field th

or

- #pods-meta-date-sidebar .pods-field th

**Comment by Shelob9** [comment]
@StaggerLeee Good catch. Can you please do a pull request against both 2.x and 3.0 for this? We'd love to have you as a contributor.

**Comment by StaggerLeee** [comment]
@Shelob, i will now.
I im not good with Github. What is difference between "issues" and "pull requests" ?
How it is handled to keep "pull requests" free from unrelated and sometimes boring support questions ?

**Comment by StaggerLeee** [comment]
Honestly, i have no clue how to "do a pull request".

Small change, put box-sizing for field too. Chrome tends to have them bit outside wrapper:

```
#side-sortables .form-table tr.pods-field > td {
box-sizing: border-box;
display: block;
line-height: inherit;
padding: 0;
margin: 0;
}
```

**Comment by Shelob9** [comment]
@StaggerLeee A pull request is a request to make a change to a repository. The simplest way to do it is to find the file in question using the web interface, and click the "pencil" edit button.

**Comment by StaggerLeee** [comment]
@Shelob9 thanks. I would do it yesterday but spen much time testing something different. And anything i tried could not get it to work. Maybe something trivial and easy but i could not do it.

There will be real life scenarios when people need for one field label above, for other field label inline. There are many different field, such Date or Time, they ususally have small input fields, dont need to be 100% width.

I did this in code.php and text.php:
```
self::$type . '_label_align' => array(
                'label' => __( 'Label Align', 'pods' ),
                'default' => 'top_aligned',
                'type' => 'pick',
                'data' => array(
                    'top_aligned' => __( 'Top aligned', 'pods' ),
                    // 'medium' => __( 'Medium', 'pods' ),
                    'left_aligned' => __( 'Left aligned', 'pods' )
                )
            ),
```

Same code in both, new option for field shows, is saved in database and holds saving.

![2015-03-17_102914](https://cloud.githubusercontent.com/assets/7382616/6684328/8649e35e-cc90-11e4-8879-a09cff3b41ad.jpg)

But could not manage to show this value in backend metabox. Could not even show text field value, here is select (pick).

How to show, echo, those field options values in Metabox ?
I see "class" and "description" do it, Tried to imitate what them do, but it is so spread around different files.

With this people could just simply decide by themselves what field will have label above field, and what field label inline field (backend only of course).

It could add dynamic classes to field wrapper and with CSS decide if display:block or display:inline, with rest of CSS properties.

Same way as this part in PodsMeta.php do:
```
<tr class="form-field pods-field pods-field-input <?php echo 'pods-form-ui-row-type-' . $field[ 'type' ] . ' pods-form-ui-row-name-' . PodsForm::clean( $field[ 'name' ], true ); ?> <?php echo $depends; ?>">
```

**Comment by shelob** [comment]
Hi guys! Please don't add me to CC in your letters :) I'm "shelob" - NOT "shelob9". Its really interesting project but I have my own ones and your requests blow my mind...

**Comment by sc0ttkclark** [comment]
@shelob let us know when you're ready to become a full contributor :)

**Comment by pglewis** [comment]
This should be fixed in 2.7.
